lib/features/postings/postings_repository.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:instreal/features/postings/posting_entity.dart';
+import 'package:instreal/features/postings/posting_firestore_model.dart';
+import 'package:instreal/features/postings/postings_firestore.dart';
+
+abstract interface class PostingsRepository {
```

Comment:
Why do you use abstract class for a feature repository? Do we need to implement it another time?

Reply:
I'm not sure what you mean. This is basically for testability. `interface` + DI pattern makes it available for unit testing.